components/input/Input.js

```diff
@@ -116,13 +116,11 @@ export default class Input extends Component {
   renderHelpText() {
     const { helpText, inverse } = this.props;
 
-    if (helpText) {
-      return (
-        <TextSmall color={inverse ? 'white' : 'neutral'} marginTop={1} soft>
-          {helpText}
-        </TextSmall>
-      );
-    }
+    return helpText && (
```

Comment:
PP: I think the return statement is superfluous here. If helpText is not falsy then it will return what comes after it.

Reply:
Nevermind, it's in a function so idd you *do* have to return it :) 

 :ok: by @InstaK on 13/02/2018 12:08:46